docs/starters.yml

```diff
@@ -1609,3 +1609,14 @@
     - Progressive Web app
     - Offline Support
     - RSS Feed
+- url: https://ueno.co/
+  repo: https://github.com/ueno-llc/ueno-gatsby-starter
+  description: Ueno's starter template for Gatsby
+  tags:
+    - TypeScript
+  features:
+    - GraphQL hybrid
+    - SEO friendly
+    - GSAP ready
+    - Nice devtools
```

Comment:
I'm a little confused by this. Is this this devtools setup? What exactly is this doing?

https://github.com/ueno-llc/ueno-gatsby-starter/blob/master/src/components/devtools/GridOverlay.tsx

Reply:
This is one of the devtools, we forgot to add our gasp tools, we will push an update tomorrow.

This particular devtools displays an overlay grid that uses configuration from scss styles folder. It has they same keyboard shortcuts as in Sketch.